05-classification.Rmd

```diff
@@ -47,16 +48,18 @@ prediction are called a **training set**. We call them a "training set" because
 we use these observations to train, or teach, our classifier so that we can use
 it to make predictions on new data that we have not seen previously.
```

Comment:
Here we should also make it explicit we have no label for the data we are trying to predict on.
```suggestion
it to make predictions on new data that we have not seen previously, and that we have no class label for.
```

Reply:
edited